### PR TITLE
fix: set menu items spacing to equal

### PR DESCRIPTION
**Pull request by Nayananga**

### CODE REVIEW

**Comment by zregvart** [comment]
@Nayananga I think we adjust the `padding-right` to `1.5rem` for the `navbar-item` to get the same effect here:

https://github.com/apache/camel-website/blob/06ff2e55d43df4d654b96be2da39354ddb8dd076/antora-ui-camel/src/css/header.css#L130-L133

File: antora-ui-camel/src/css/header.css [hunk 1]
```diff
@@ -130,6 +130,8 @@ body {
 .navbar-item {
   flex: none;
   margin-bottom: 1px;
+  font-weight: bolder;
```
Comment (Nayananga) [inline antora-ui-camel/src/css/header.css]: this made menu items in burger menu more visible

Comment (zregvart) [reply antora-ui-camel/src/css/header.css]: This is overridden here:

https://github.com/apache/camel-website/blob/fb0ff7bbfe19b2b98d7be22d158a3676c55dc85d/antora-ui-camel/src/css/nav.css#L187

So it doesn't have an effect I'm afraid. Also would be good to use variables so we can tweak from vars.css.

**Comment by zregvart** [comment]
Perfect, thanks!

**Pull request merged**